stable/nuclio/README.md

```diff
@@ -0,0 +1,76 @@
+# Nuclio
+
+##  High-Performance Serverless Event and Data Processing Platform
+
+[Nuclio](https://nuclio.io) is a new "serverless" project, derived from [iguazio](https://iguazio.com)'s elastic data life-cycle management service for high-performance events and data processing
+
+## QuickStart
+
+```bash
+$ helm install stable/nuclio --name foo --namespace bar
+```
+
+## Introduction
+
+This chart bootstraps a Nuclio deployment (controller and playground) and service on a Kubernetes cluster using the Helm Package manager.
+
+## Prerequisites
+
+- Kubernetes 1.7+
```

Comment:
We officially support the current and previous minor version of Kubernetes, which would be 1.8+.

Reply:
These are the prerequisites for nuclio - no point in changing it every release Kubernetes has.
1.7+ includes 1.8.